articles/container-apps/tutorial-event-driven-jobs.md

```diff
@@ -76,6 +90,32 @@ The job uses an Azure Storage queue to receive messages. In this section, you cr
         --connection-string "$QUEUE_CONNECTION_STRING"
     ```
 
+## Create a user-assigned managed identity
+
+You can pull images from private repositories in Microsoft Azure Container Registry using managed identities for authentication to avoid the use of administrative credentials. When possible, you should use a user-assigned managed identity to pull images.
+
+1. Create a user-assigned managed identity. Before you run the following commands, replace the `\<PLACEHOLDERS\>` with the name of your managed identity. 
+
+    ```bash
+    IDENTITY="<YOUR_IDENTITY_NAME>"
```

Comment:
How will the reader know what value to put here? Add a comment to tell them they can pick a name.

Reply:
Added.

Reply:
Added.